### ISSUE TITLE
[Accessibility] A11y_Manage.IIS.Net_Web Server_Modules_Keyboard: Unable to access interactive tabs "Managed" & "Native" using keyboard

### ISSUE THREAD

**Issue by v-supupp** [Microsoft/IIS.WebManager#88]
**Environment Details:**
URL: https://manage.iis.net/webserver/modules
Browser Details: Microsoft Edge 42.17134.1.0, Google Chrome Version 68.0.3440.106
**WCAG ID Violated:** WCAG2.1.1

GitHubTags: #A11yTCS; #A11yMAS; #DesktopWeb; #Win10; #A11y_Manage.IIS.Net; #Keyboard; #C+E-H; #.NETWebsites; #Chrome; #Edge; WCAG2.1.1

**Repro Steps:**
1.	Download and install "Microsoft IIS Administration" application.
2.	Enable Internet information Services.
3.	Open "Microsoft IIS" application using the URL: https://manage.iis.net/
4.	Tab till Web Server and press enter
5.	Tab till Modules and press enter
6.	Validate that all the interactive controls are accessible using keyboard.

**Actual Result:**
Unable to access interactive controls “Managed” and “Native” using keyboard.

**Expected Result:**
We should be able o access interactive controls “Managed” and “Native” using keyboard.

**Note:**
This issue is observed in all the pages having interactive tabs

**User Impact:**
Keyboard only users finds its difficult in working through the application if interactive controls are not accessible using keyboard.

**Attachments:**
[Bug 88_Keyboard_InteractiveControls.pptx](https://github.com/Microsoft/IIS.WebManager/files/2318454/Bug.88_Keyboard_InteractiveControls.pptx)

**Comment by jhkimnew** [comment]
Fixed

**Comment by Roopesh-T** [comment]
**GitTags**: #A11ySev1;

**Comment by Yashi1201** [comment]
#Closed;
Manage.iis.net is deprecated. The "Manage IIS" button on the www.iis.net home page will be removed.
